src/actions/timelineAction.jsx

```diff
@@ -66,7 +66,7 @@ export const addNote = (noteText, incidentId) => {
     return axios
       .post(notesUrl, {
         note: noteText,
-        userEmail: localStorage.getItem('email')
+        userId: localStorage.getItem('email')
```

Comment:
Why this change?

Reply:
There was a bug. The mockController/middleware was expecting `userId`, but the action was sending it `userEmail`

Reply:
@peterpaints Has this been tested with the actual backend?

Reply:
I've removed the change. I've seen the error of my ways.